src/ng/directive/ngOptions.js

```diff
@@ -446,12 +446,12 @@ var ngOptionsDirective = ['$compile', '$document', '$parse', function($compile,
       };
 
       var removeEmptyOption = function() {
+        emptyOption.removeAttr('selected');
```

Comment:
Minor (as it isn't expected to happen often in an app's lifecycle), but this could be moved to an `else` branch (since we only care about the attribute if we aren't removing the element).